File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    	sort(A.rbegin(), A.rend());
	sort(B.rbegin(), B.rend());

	auto Ait = A.begin();
	auto Bit = B.begin();

	while (Ait != A.end() && Bit!=B.end())
	{		
		if (*Ait < *Bit)
		{
			answer++;
			Ait++;
			Bit++;
		}
        else
			Ait++;
	}
    return answer;
}

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.cs
using System;
using System.Linq;
public class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        Array.Sort(A);
        Array.Sort(B);
        int k = B.Length-1;
        for(int i = A.Length-1;i>=0;i--)
        {
            if(A[i] < B[k])
            {
                answer++;
                k--;
            }
            else
            {
                continue;
            }
        }
        return answer;
    }
}